resources/assets/js/effect/fxraket.ts

```diff
@@ -0,0 +1,44 @@
+// importeer stijl
+import '../../sass/effect/raket.scss';
+
+class UVo {
+
+	private static startAnimatie() {
+		if (!document.body.classList.contains('scrolling')) {
+			document.body.classList.add('scrolling');
+		}
+	}
+
+	private static stopAnimatie() {
+		if (document.body.classList.contains('scrolling')) {
+			document.body.classList.remove('scrolling');
+		}
+	}
+
+	public start() {
+		window.addEventListener('scroll', this.ontstuiter(UVo.startAnimatie, 250, true), {passive: true});
+		window.addEventListener('scroll', this.ontstuiter(UVo.stopAnimatie, 250, false), {passive: true});
+	}
+
+	private ontstuiter(func: any, wait: number, immediate: boolean) {
+		let timeout: number | undefined;
+		return function () {
```

Comment:
Kan je deze logica niet beter naar util verplaatsen? Ik heb niet in een oogopslag het idee wat er gebeurt.

Reply:
Verplaatst ✔